### ISSUE TITLE
Mypy giving error when using emojize

### ISSUE THREAD

**Issue by cap-jmk** [carpedm20/emoji#250]
When using emojize inside a typed function, I recieve the following error: 
```bash
error: Call to untyped function "emojize" in typed context  [no-untyped-call]
```
Do you know how to handle that?

**Comment by cvzi** [comment]
Are you using the latest version of the package? There was a bug in earlier version that was fixed in #239

**Comment by MQSchleich** [comment]
Yes, I am using v2.2.0

**Comment by cvzi** [comment]
I am not familiar with `lia` what does it do?

**Comment by cvzi** [comment]
I found the problem, the `.tar.gz` package doesn't ship the type information at the moment, i.e. the `*.pyi` files aren't included. 

This will be fixed by #248

If I install from that pull request via `pip install --force-reinstall git+https://github.com/cvzi/emoji@wheel` then mypy is happy.

**Comment by cap-jmk** [comment]
> I am not familiar with `lia` what does it do?

It helps with CI. I am making frequently a lot of mistakes recently, so I code a package that helps mitigating them. It includes
1.) Linting 
2.) Automated formatting of various sorts
3.) Typechecking
4.) Git checking

And thank you for the fix :)
